common/src/main/java/com/wynntils/core/consumers/overlays/Overlay.java

```diff
@@ -63,6 +76,35 @@ protected Overlay(
         this.verticalAlignmentOverride.store(verticalAlignmentOverride);
     }
 
+    /**
```

Comment:
These javadocs seems to be a bit non-standard.  I don't think you need the <li>s, do you?

Reply:
I personally felt that having these 2 methods could be confusing, so I wanted to make the doc as clear as possible

Reply:
Adding a list or not does not change anything for understandability. Javadocs already respect newlines iirc (you may have to put 1 empty line in between content lines)